comp120-evaluation.tex

```diff
@@ -0,0 +1,57 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\usepackage{graphicx}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Reflective Report}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Reflective Report}
+
+% Please put your student number in the author field
+\author{1507290}
+
+\begin{document}
+
+\maketitle
+
+\abstract{}
+
+\section{Introduction}
+
+\section{Reading and Understanding Peers' Code}
+Due to a lack of experience in reading and writing code, I am experiencing difficulty comprehending peers' code. At this stage, the most apparent consequence of this is that it is causing me difficulty in peer review sessions, as I have often found that I struggle to construct any useful comments.
+
+If I have not developed this skill in preparation for the forthcoming collaborative projects, I will struggle to review my peers' contributions and understand the code I am contributing to. This is equally important if I am going to collaborate with others professionally as an independent games developer.
+
+In order to develop this skill, I will ensure that I read peers' code each week and attempt to predict its outcome before running the program. I will review how often I was able to accurately understand what the code would do after two months.
+
+\section{Time Management}
+My perfectionistic nature combined with being out of education and work for a long time has caused me to struggle to manage my time effectively. Rather than not spending enough time working, the most significant issue for me is spending more time than necessary perfecting a single project whilst neglecting others. I have also noticed that I get distracted by adding unnecessary features to projects. 
+
+It is crucial that I develop my ability to manage my time in preparation for the increased workload as the course progresses. Similarly, it is important that I learn when to consider something done, in order to focus on other tasks. This will also be important if I become an independent games developer, as it will allow games to be released rather than being endlessly worked on. After extensively using Trello with my Kivy project, I have noticed that it helps with this aspect. I think that this is because it breaks the project down and allows me to see when I have completed all of the tasks. It also ensures that I focus on what's in the sprint plan, rather than adding unnecessary features.
+
+I will tackle this issue by creating a flexible study plan at the beginning of each week. I will also set up a Trello board for each project and utilise sprint planning. At the end of the second term, I will assess whether this allowed me to comfortably meet assignment deadlines.
```

Comment:
I suggest, as an additional measure, using the Trello board to determine how many unnecessary features you worked on during your projects.